client/signup/steps/domains/index.jsx

```diff
@@ -775,7 +775,11 @@ class DomainsStep extends Component {
 		let isExternalBackUrl = false;
 
 		const previousStepBackUrl = this.getPreviousStepUrl();
-		const sitesBackLabelText = translate( 'Back to Sites' );
+		const sitesBackLabelText =
+			userSiteCount && userSiteCount === 1
+				? translate( 'Back to My Home' )
+				: translate( 'Back to Sites' );
+		const defaultBackUrl = userSiteCount && userSiteCount === 1 ? `/home` : `/sites`;
```

Comment:
As both expressions share the same condition, you could use a destructured array to shorten this:

```
const [ sitesBackLabelText, defaultBackUrl ] =
			userSiteCount && userSiteCount === 1
				? [ translate( 'Back to My Home' ), '/home' ]
				: [ translate( 'Back to Sites' ), '/sites' ];
```

There is also no need to use template literals for URLs here.

Reply:
Fancy! Added in b82f6bc6c8